### PR TITLE
feat/possessionProvider

### PR DESCRIPTION
**Pull request by A-Dylan-R**
Added possessionProvider

### CODE REVIEW

File: src/providers/possessionProvider.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                console.log(`delete error: ${error}`);
+                throw error;
+            }
+        } else {
+            throw new Error(`Unsupported resource: ${resource}`);
+        }
+    }
+}
+
+
```
Comment (AndyPendragon) [inline src/providers/possessionProvider.ts:73]: Use format with spaces 2 and delete all unusual white spaces

File: package.json [hunk 2]
```diff
@@ -11,6 +11,7 @@
   },
   "dependencies": {
     "axios": "^1.7.2",
+    "openapi": "^1.0.1",
```
Comment (AndyPendragon) [inline package.json:14]: we don't need openapi in our dependencies, it's just used to generate the client for a time

**Comment by A-Dylan-R** [comment]
Alright, thank you for the review


if (resource === "possessions") {
  try {
    possessionApi.deletePatrimoinePossessionByNom(nomPatrimoine,
nomPossession);
    return { success: true };
  } catch (error) {
    console.log(`delete error: ${error}`);
    throw error;
  }
} else {
  throw new Error(`Unsupported resource: ${resource}`);
}


like this then ?

**Review by AndyPendragon — CHANGES_REQUESTED** [review]
1. Format it (with prettier ?) spaces: 2
2. Remove whitespaces
3. Do we need OpenAPI in dependencies? I don't think so.

Beyond that, the code looks good and is easy to read.
Good job!

**Pull request merged**